components/elm/bld/namelist_files/use_cases/2010_CMIP6_control.xml

```diff
@@ -11,8 +11,8 @@
 <stream_year_first_ndep phys="elm" use_cn=".true." ndepsrc="stream" >2010</stream_year_first_ndep>
 <stream_year_last_ndep  phys="elm" use_cn=".true." ndepsrc="stream" >2010</stream_year_last_ndep>
 
-<stream_year_first_pdep phys="elm" use_cn=".true." ndepsrc="stream" >2010</stream_year_first_pdep>
-<stream_year_last_pdep  phys="elm" use_cn=".true." ndepsrc="stream" >2010</stream_year_last_pdep>
+<stream_year_first_pdep phys="elm" use_cn=".true." ndepsrc="stream" >2000</stream_year_first_pdep>
+<stream_year_last_pdep  phys="elm" use_cn=".true." ndepsrc="stream" >2000</stream_year_last_pdep>
```

Comment:
Same comment as https://github.com/E3SM-Project/E3SM/pull/6113/files#r1440056541.

Reply:
Good point, @bishtgautam . Intuitively, when first testing it, did try 2010. But look like only simyr2000 stream file is available, which is used for piControl as well, so I thought it being treated as time-invariant. If time varying for this data stream is necessary, we will need quick help to generate the files for compsets of different TIME (or forcing scenarios)
The file is : `inputdata/lnd/clm2/pdepdata/fpdep_clm_hist_simyr2000_1.9x2.5_c150929.nc`